### PR TITLE
Ex 4

### PR DESCRIPTION
**Pull request by hennigk**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,46 +1,9 @@
 <!doctype html>
 <html>
     <head>
+        <link rel="stylesheet" href="css/main.css">
+        <link rel="stylesheet" href="css/normalize.css">
```
Comment (nyanofthemoon) [inline index.html]: @hennigk Reset and normalize stylesheets always come before your own CSS files. Please fix.

File: index.html [hunk 2]
```diff
@@ -75,6 +38,14 @@
         
         <section class="schedule">
             <h2>Schedule</h2>
+            <table>
+                <tr>
+                <th>Morning</th><th>Afternoon</th>
```
Comment (nyanofthemoon) [inline index.html]: @hennigk Please fix identation of the table header.

**Comment by ziad-saab** [comment]
@hennigk can you please fix the ordering of your included CSS files?

**Comment by hennigk** [comment]
@ziad-saab - I changed the ordering of the style sheets

**Comment by ziad-saab** [comment]
:+1:

**Pull request closed without merging**